exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/writer/TestParquetWriter.java

```diff
@@ -780,17 +780,31 @@ public void testImpalaParquetBinaryAsVarBinary_DictChange() throws Exception {
   Test the reading of a binary field as drill timestamp where data is in dictionary _and_ non-dictionary encoded pages
    */
   @Test
-  @Ignore("relies on particular time zone, works for UTC")
   public void testImpalaParquetBinaryAsTimeStamp_DictChange() throws Exception {
     try {
       testBuilder()
-          .sqlQuery("select int96_ts from dfs.`parquet/int96_dict_change` order by int96_ts")
+          .sqlQuery("select min(int96_ts) date_value from dfs.`parquet/int96_dict_change`")
```

Comment:
Did you try WHERE statement?

Reply:
I did not try a WHERE statement, MIN was used to select a single record to compare. Was there any specific reason to use WHERE?

Reply:
It is just more obvious what result is expected. But using MIN is ok.